.gitattributes

```diff
@@ -1,5 +1,6 @@
 /.github/                       export-ignore
 /.phive/                        export-ignore
+/test/                          export-ignore
```

Comment:
When you work on a package and declare test code in a separate directory, adding exclusions for test code in `.gitattributes` is less work.